tests/test_pipeline.cpp

```diff
@@ -181,135 +179,6 @@ TEST_F(TestPipeline, TwoSegment)
     LOG(INFO) << "Done" << std::endl;
 }
 
-/*
```

Comment:
Are these not relevant anymore, or just don't work?

Reply:
Correct, we no longer support manual tagging.